shell/run_shell_cmd.go

```diff
@@ -30,19 +32,31 @@ func RunTerraformCommandAndCaptureOutput(terragruntOptions *options.TerragruntOp
 func RunShellCommand(terragruntOptions *options.TerragruntOptions, command string, args ...string) error {
```

Comment:
What I'm thinking is that *all* functions should return the output, regardless of caller. The auto-retry will quite heavily depend on that and by looking at the whole code base there aren't many cases where the output is *not* needed. As such, I would be tempted to rename some of the functions - for example RunShellCommandAndCaptureOutput to something like RunShellCommandMuted. And pipe the stdout/err to ioutil.Discard instead, as the output is captured with the RunShellCommand anyways. Opinions?